lib/common/hook.js

```diff
@@ -64,9 +67,28 @@ function hookFactory(
         });
     };
 
-    function _isQualifiedData(data, filters) {
-        //TODO: design filters for hook here.
-        return true;
+    function _filter(data, filters) {
```

Comment:
suggest to add some comments for these two new functions for reader's better understanding, about the filter's structure, or why it is an OR logic while the other is an AND logic.